c2corg_ui/static/partials/cards/outings.html

```diff
@@ -8,12 +8,9 @@
 
     <span class="list-item-title">{{locale.title}}</span>
     <span class="list-item-lang" ng-if="cardCtrl.lang != locale.lang">({{locale.lang}})</span>
-
     <br>
-    <span class="quality" ng-if="doc['quality']" uib-tooltip="{{'quality' | translate}}">
-      <span class="glyphicon glyphicon-file"></span>
-      <span x-translate>{{doc['quality']}}</span>
-    </span>
+    <div class="quality" ng-if="doc['quality']" ng-class="doc['quality']"
+         uib-tooltip="{{'quality'| translate}} : {{doc['quality'] | translate}}" tooltip-placement="left"></div>
```

Comment:
When you run ``node tools/extract-messages.js c2corg_ui/static/partials/cards/outings.html > .build/locale/c2corg_ui-client.pot`` does `` .build/locale/c2corg_ui-client.pot`` contains ``doc['quality']`` (just to make sure it is not extracted because of the translate directive)?